dev-client/src/screens/HomeScreen.tsx

```diff
@@ -29,22 +28,17 @@ const HomeView = () => {
     [dispatch],
   );
   return (
-    <VStack>
-      <Box flexBasis="90%">
-        <SiteMap
-          updateUserLocation={updateUserLocation}
-          sites={sites}
-          center={mapCenter}
-        />
-        <BottomSheet
-          sites={sites}
-          showSiteOnMap={site => setMapCenter([site.longitude, site.latitude])}
-        />
-      </Box>
-      <Box flexBasis="10%">
-        <BottomNavigation />
-      </Box>
-    </VStack>
+    <ScreenScaffold>
```

Comment:
note: i created a `ScreenScaffold` component to just do the wrapping of screens in a bottom sheet, since we were repeating that logic everywhere. perhaps there will be other useful parameters or logic for the screen scaffold to perform one day, perhaps not!

Reply:
Yes, good idea, I was thinking the same thing recently.